docs/data-ingestion/_index.md

```diff
@@ -0,0 +1,26 @@
+---
+# sort: 14
+title: Data Ingestion 
+weight: 0
+menu:
+  docs:
+    parent: 'victoriametrics'
+    identifier: 'data-ingestion'
+    weight: 7
+aliases:
+- /data-ingestion.html
+- /data-ingestion.html
+- /dataingestion/
+---
```

Comment:
Duplicated alias: `/data-ingestion.html`?

Reply:
Changed so it is 
```
- /data-ingestion.html
- /Data-Ingestion.html
- /dataingestion/
```